charts/pro-ui/templates/ui/styles-import-starter.yaml

```
@@ -0,0 +1,64 @@
+{{ if .Values.stylesImporter.startOnDeploy }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "pro.ui.styles-importer-name" . }}-starter
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  backoffLimit: {{ .Values.stylesImporter.backoffLimit }}
+  template:
+    spec:
+      {{- with .Values.stylesImporter.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      restartPolicy: Never
+      serviceAccountName: {{ include "pro.ui.service-account-name" . }}
+      volumes:
+        - name: temp-volume
+          emptyDir: {}
+      containers:
+        - name: {{ include "pro.ui.styles-importer-name" . }}
+          image: {{ required "A valid .Values.dgctlDockerRegistry entry required" $.Values.dgctlDockerRegistry }}/{{ .Values.stylesImporter.image.repository }}:{{ .Values.stylesImporter.image.tag }}
+          imagePullPolicy: IfNotPresent
+          resources:
+            {{- toYaml .Values.stylesImporter.resources | nindent 16 }}
+          volumeMounts:
+            - mountPath: "/tmp"
+              name: temp-volume
+          env:
+            - name: JOB_NAME
+              value: {{ include "pro.ui.styles-importer-name" . }}
+            - name: ES_HOST
+              value: http://some_fake_url
+            - name: API_URL
+              value: http://some_fake_url
+            - name: MODE
+              value: "ManifestData"
+            - name: manifest_filename
+              value: {{ .Values.dgctlStorage.manifest }}
+            - name: manifest_section
+              value: "pro-ui"
+            - name: manifest_data_uncompress_file_pattern
+              value: "*.tar.gz"
+            - name: S3Settings__Url
+              value: {{ .Values.dgctlStorage.host }}
+            - name: S3Settings__Secure
+              value: false
```

```suggestion
              value: "false"
```

Иначе ошибка при деплои:

> Job in version "v1" cannot be handled as a Job: json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string

Но как быть если бакеты предоставляются по https ?
Должна быть опция:
```suggestion
              value: "{{ .Values.dgctlStorage.secure }}"
```